src/main/java/com/amazon/opendistroforelasticsearch/security/rest/DashboardsInfoAction.java

```diff
@@ -94,8 +94,8 @@ public void accept(RestChannel channel) throws Exception {
                     builder.field("user_name", user==null?null:user.getName());
                     builder.field("not_fail_on_forbidden_enabled", evaluator.notFailOnForbiddenEnabled());
                     builder.field("kibana_mt_enabled", evaluator.multitenancyEnabled());
```

Comment:
It appears that these field names have been changed in the security-dashboards-plugin - https://github.com/opensearch-project/security-dashboards-plugin/blob/main/server/backend/opendistro_security_plugin.ts#L40-L42

Can we make sure field names are consistent?

Reply:
This is in the scope of another migration task for Kibana that @hsiang9431-amzn is going to take care of.